Basic/APIDiorama/README.md

```diff
@@ -0,0 +1,15 @@
+# API Diorama
+
+This is a sample project that provides a series of scenes. Each scene aims at explaining a specific API or system commonly used in Netcode for GameObjects. In this sample you will learn more about client-server communication, state synchronization, and other typical mechanics of multiplayer games.
+
+## Sample feature
+
+Making a multiplayer game is different than making a singleplayer one. 
+
+Understanding how to send data and events over the network and to the right players is a challenge.
+
+API Diorama's aim is to show you how to perform common actions in a multiplayer environment, so that you can build the features of your game with them in mind.
+
+In this sample you'll learn about the core features of Netcode For GameObjects ("NGO") through practical examples that you'll be able to extend and adapt to fit the gameplay logic of your own game. 
```

Comment:
```suggestion
In this sample you'll learn about the core features of Netcode For GameObjects (Netcode) through practical examples that you'll be able to extend and adapt to fit the gameplay logic of your own game. 
```

Reply:
since this is a protected branch, I added this suggestion as part of another PR